dts/arm/nxp/nxp_mcxn94x_common.dtsi

```diff
@@ -582,6 +582,11 @@
 			/* MCXN94x ROM Flash API supports writing of 128B pages. */
 			write-block-size = <128>;
 		};
+
+		uuid: flash@1100000 {
```

Comment:
node name says flash?

Reply:
The uuid is in the flash. Similar to some LPC devices like LPC55S69.

Reply:
I don't think the uuid is erasable, which is a defining feature of a flash memory, that's what the name comes from, so don't think the node name flash makes sense here

Reply:
I have updated the node name.